### ISSUE TITLE
Make Glathida files download work in parallel

### ISSUE THREAD

**Issue by JordiBolibar** [ODINN-SciML/Sleipnir.jl#39]
Right now the downloading of Glathida files with a `pmap` doesn't work. This should be fixed in the future in order to speed up the downloading of data for lots of glaciers.

**Comment by albangossard** [comment]
I'm not sure this issue is still a thing.

Currently in the code we use `filter_missing_glaciers!(rgi_ids::Vector{String}, params::Parameters)` (and not `filter_missing_glaciers!(glaciers::Vector{Glacier2D}, params::Parameters)` which by the way could be removed) and this downloads one CSV for all the glaciers. There is no need of parallelism.

Additionally, since glathida data are now preprocessed with Gungnir, the function `get_glathida_path_and_IDs` is no longer used and I think that it can be removed.

So to summarize we should remove `get_glathida_path_and_IDs` and one of the implementation of `filter_missing_glaciers!`. @JordiBolibar do you agree on this conclusion?

**Comment by JordiBolibar** [comment]
Yes, this is again legacy code. At the beginning we were doing a manual implementation of the downloading of Glathida. Later, this has been included officially in OGGM, so it's much easier.

**Comment by albangossard** [comment]
Thank for this precision! I took profit of https://github.com/ODINN-SciML/Sleipnir.jl/pull/79/ to remove the two aforementioned functions.
